influencer/src/components/organisms/Dashboard/DashboardContainer.tsx

```diff
@@ -0,0 +1,49 @@
+'use client';
+import { useState } from 'react';
+// import DashboardCardData from '../hooks/use-dashboard-data'; // Temporarily disabled - hook not merged
+
```

Comment:
This hook is ready to use now. why is still commented out?? 
you need to get data from hook and render it in your container